surround/cli.py

```diff
@@ -33,7 +33,7 @@
             "tests",
         ],
         "files": [
-            ("requirements.txt", "surround=={version}\ntornado==6.0.1"),
```

Comment:
Generated projects should have tornado as a dependency.

Reply:
No, we don't need it. The requirements file is only read by docker and it does not need tornado!